Example/src/TagInputExample.js

```diff
@@ -39,6 +64,39 @@ export default class TagInputExample extends Component {
             maxHeight={75}
           />
         </View>
+
+        <Text style={{marginVertical: 10}}>Horizontal Scroll</Text>
+        <View style={{marginBottom: 10, flexDirection: 'row', alignItems: 'center', backgroundColor: 'lightblue'}}>
+          <Text>To: </Text>
+          <TagInput
+            ref='horizontalTagInput'
```

Comment:
Notwithstanding my comment above, string values from the ref parameter are discouraged in favor of using ref functions, which typecheck much better.